### PR TITLE
Bring hello-express up-to-date

### PR DESCRIPTION
**Pull request by potch**
* Do something with the server besides serve static files
* Use modern idiomatic ES6
* Use modern web API features
* streamline and modernize styles

### CODE REVIEW

**Comment by potch** [comment]
You can see these changes live here: https://chalk-duke.glitch.me/

File: public/script.js [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    // iterate through every dream and add it to our page
+    dreams.forEach(appendNewDream);
+  
+    // listen for the form to be submitted and add a new dream when it is
+    dreamsForm.addEventListener("submit", event => {
+      // stop our form submission from refreshing the page
+      event.preventDefault();
+
+      // get dream value and add it to the list
+      let newDream = dreamsForm.elements.dream.value;
```
Comment (keithk) [inline public/script.js]: I know we don't have multiple forms on the page, but would it make more sense to use
`event.target.elements.dream.value` here to get the value of the submitted form?

File: README.md [hunk 2]
```diff
@@ -1,27 +1,30 @@
...
--------------------
 
-\ ゜o゜)ノ
+## Made by [Glitch](https://glitch.com/)
+
+**Glitch** is the friendly community where you'll build the app of your dreams. Glitch lets you instantly create, remix, edit, and host an app, bot or site, and you can invite collaborators or helpers to simultaneously edit code with you.
+
+Find out more [about Glitch](https://glitch.com/about).
+
+( ᵔ ᴥ ᵔ )
```
[the remark this answers is not in the document]
Comment (potch) [reply README.md]: I liked the idea of the different apps having different kaomoji faces!

File: public/script.js (hunk 1 above)
Comment (potch) [reply public/script.js]: I was thinking something similar! I figured I had to have a reference to the form either way to attach a listener, and so if I have it I may as well use it.

File: public/style.css [hunk 3]
```diff
@@ -1,71 +1,56 @@
...
   color: #373fff;
-}
-
-.bold {
-  font-weight: bold;
-}
-
-p {
-  max-width: 600px;
+  max-width: calc(100% - 5rem);
```
Comment (smonette) [inline public/style.css]: :chefs kiss:

File: server.js [hunk 4]
```diff
@@ -1,22 +1,34 @@
 // server.js
 // where your node app starts
 
-// init project
+// we've started you off with Express (https://expressjs.com/)
+// but feel free to use whatever libraries or frameworks you'd like through `package.json`.
```
Comment (smonette) [inline server.js]: (Probably not relevant to this PR, so feel free to ignore) I always feel like this comment around frameworks and libraries needs to be elaborated on. Adding a templating language needs updates to `package.json` AND setting a view engine in `server.js` which might not be obvious.

File: README.md [hunk 5]
```diff
@@ -1,27 +1,30 @@
-Welcome to Glitch
-=================
+# hello-express
 
-Click `Show` in the header to see your app live. Updates to your code will instantly deploy and update live.
+A webpage and a server that sends it data
```
Comment (melissamcewen) [inline README.md]: should it be a server that serves a webpage or something like that? it makes it sound a bit seperate

File: public/style.css [hunk 6]
```diff
@@ -1,71 +1,56 @@
-/* styles */
-/* called by your view template */
+/* styles pulled in by index.html */
```
Comment (melissamcewen) [inline public/style.css]: should it be more along the lines of saying this file is imported into index.html and styles the page?

**Comment by melissamcewen** [comment]
Looks awesome thanks for updating this!

File: README.md (hunk 5 above)
Comment (potch) [reply README.md]: I like this idea! will update.

File: public/style.css (hunk 6 above)
Comment (potch) [reply public/style.css]: I updated this comment as well!

File: server.js (hunk 4 above)
Comment (potch) [reply server.js]: once hello-express is updated, I plan on making a template engine variation and I can reference it here!

**Pull request merged**